Exercise-24/index.js

```diff
@@ -0,0 +1,112 @@
+"use strict"
+//for loop - adding up nubmers 0-1000 break when result is more than 400
+
+let var1 = 0;
+let var2 = 0;
+for (var2 = 0; var2 <= 1000; var2++) {
+    var1 += var2;
+    console.log(var1);
+
+    if (var1 > 400) {
+        break;
+    }
+}
+
+//while loop - output to console 0-10
+let firstWhile = 0;
+
+while (firstWhile <= 10) {
+    console.log(firstWhile);
+    firstWhile++;
+}
+
+//while loop- output to console 100-0
+let secondWhile = 100;
+
+while (secondWhile >= 0) {
+    console.log(secondWhile);
+    secondWhile--;
+}
+
+//while loop- output to console even numbers 0-100
+let thirdWhile = 0;
+
+while (thirdWhile <= 100) {
+    console.log(thirdWhile);
+    thirdWhile++;
+    if (thirdWhile % 2 === 1) {
+        continue;
+    }
+}
+
+//do loop - output to console 0-10
+let firstDo = 0;
+
+do {
+    console.log(firstDo);
+} while (firstDo++ <= 10);
+
+//do loop- output to console 100-0
+let secondDo = 100;
+
+do {
+    console.log(secondDo);
+} while (secondDo-- >= 0);
```

Comment:
-1 is logged in this case. Please revisit the code. It would be more readable to decrement the variable outside the condition